### ISSUE TITLE
Issue with some mekanism items

### ISSUE THREAD

**Issue by mllightitup** [YaLTeR/MouseTweaks#33]
Please read this first: https://github.com/mekanism/Mekanism/issues/6401

I thought it was a mekanism bug and I wrote them about it, but in the end it turned out to have something to do with mouse tweaks

**Comment by YaLTeR** [comment]
Could you try with LMB tweak with and without item disabled in Mouse Tweaks config file?

**Comment by mllightitup** [comment]
> Could you try with LMB tweak with and without item disabled in Mouse Tweaks config file?

LMBTweakWithItem=0
LMBTweakWithoutItem=0

with these settings, everything became as it should be

**Comment by YaLTeR** [comment]
Could you try turning one of them on to see which one it is that affects it?

**Comment by mllightitup** [comment]
> Could you try turning one of them on to see which one it is that affects it?

with these settings everything is fine

LMBTweakWithItem=1
LMBTweakWithoutItem=0

**Comment by YaLTeR** [comment]
If I understood correctly, you're having issues when just Shift-LMB clicking in a Mekanism GUI. However, in theory Mouse Tweaks should not affect this as it doesn't cancel any events as part of the LMB mechanic, it only [sends a shift-left-click](https://github.com/YaLTeR/MouseTweaks/blob/9ed9f89db75c3bb82cd5f63a1aaf71a0553ee67b/src/main/java/yalter/mousetweaks/Main.java#L242) when you drag across a new slot.

**Comment by mllightitup** [comment]
Everything that happens is like in this video: https://www.youtube.com/watch?v=Lp6qmE4ZAFo
On each item I click with held shift, buy some items like "resistive heater" which contains energy, it does not work. 

I've just double-checked everything and realized that these settings do not affect anything. Most likely I just forgot to charge the resistive heater and and went to answer you faster.

When only mekanism is installed without other mods this problem is not present, once I install mouse tweaks the problem appears. Changing these two parameters does not help. 
Sorry for confusing

**Comment by YaLTeR** [comment]
Okay, well, that's still odd as Mouse Tweaks shouldn't mess with anything unless it's an RMB drag.

**Comment by mllightitup** [comment]
> Okay, well, that's still odd as Mouse Tweaks shouldn't mess with anything unless it's an RMB drag.

If no one can confirm that this is a bug, then most likely this is my personal problem or I do not understand something.
I just can't figure out how it works. It is unstable and works differently in every situation. Yesterday I rechecked everything and everything worked as I wrote above. Today everything is different. I do not know what is this

I will ask a friend to do it today and, if confirmed, I will write here
